com.unity.formats.fbx/Editor/Sources/Recorders/FbxRecorder/FbxRecorderSettings.cs

```diff
@@ -229,6 +249,12 @@ string GameObjectSceneNameResolver(RecordingSession session)
             return go != null ? go.scene.name : "None";
         }
 
+        /// <summary>
+        /// Indicates if the current platform is supported (True) or not (False).
+        /// </summary>
```

Comment:
tried to use `/// <inheritdoc/>` here, but it didn't seem to work when I used the doc package to test the docs